consensus/vbft/msg_builder.go

```diff
@@ -221,6 +221,28 @@ func (self *Server) constructBlock(blkNum uint32, prevBlkHash common.Uint256, tx
 	return blk, nil
 }
 
+func (self *Server) constructCrossChainMsg(blkNum uint32) (*types.CrossChainMsg, error) {
+	root, err := self.chainStore.getCrossStatesRoot(blkNum)
+	if err != nil {
```

Comment:
call chainStore function should use  self.blockPool  call

Reply:
fixed